habitat/datasets/rearrange/rearrange_generator.py

```diff
@@ -933,39 +933,40 @@ def get_config_defaults() -> CN:
             logger.info(f" SceneDataset: {mm.active_dataset}\n")
```

Comment:
Maybe too large of a change, but I think it would make more sense to construct the message and then log it all at once rather than logging it piece by pieces.

Reply:
Agreed this would be better. You are welcome to change it here or wait for a later PR.